telegram/ext/conversationhandler.py

```diff
@@ -124,7 +130,8 @@ def __init__(self,
                  timed_out_behavior=None,
                  per_chat=True,
                  per_user=True,
-                 per_message=False):
+                 per_message=False,
+                 conversation_timeout=0):
```

Comment:
I prefer `None` as the default value for no timeout

Reply:
I'll change it, but there is no difference, I think.